zebra/zebra_mpls.c

```diff
@@ -1323,6 +1323,9 @@ static zebra_nhlfe_t *nhlfe_add(zebra_lsp_t *lsp, enum lsp_types_t lsp_type,
 	nhlfe = nhlfe_alloc(lsp, lsp_type, gtype, gate, ifindex, num_labels,
 			    labels);
 
+	if (!nhlfe)
```

Comment:
@rwestphal I am not sure this is the correct long term way to approach this problem.  we should prevent zapi messages that pass in a blackhole nexthop and maybe add a debug log for the nhlfe_alloc that this is a dev escape?